habana/1.10.0/ubi8-python-3.8/Pipfile

```diff
@@ -45,18 +45,24 @@ scipy = "~=1.10.1"
 jupyter-bokeh = "~=3.0.5"
 jupyterlab-lsp = "~=3.10.2"
 jupyterlab-widgets = "~=3.0.5"
-jupyter-resource-usage = "~=0.6.0"
+jupyter-resource-usage = "~=0.7.2"
 
 # Parent image requirements to maintain cohesion
 jupyterlab = "~=3.5.3"
 jupyter-server = "~=2.1.0"
-jupyter-server-proxy = "~=3.2.2"
+jupyter-server-proxy = "~=3.2.3"
```

Comment:
The fully fixed release is 3.2.4, but since the `~=` is used, it is installed anyway, see https://quay.io/repository/opendatahub/workbench-images/manifest/sha256:d6c44c0bc78a4fd4fd272f1edfa44924d73ffd971ce17ed884700dcef762808c?tab=packages